docs: modify core dump (#3585)
spectrocloud/librarium: docs/docs-content/clusters/edge/local-ui/host-management/download-files.md

---

> sidebar_label: "Download Files from Local II"

Reviewer: ```suggestion
sidebar_label: "Download Files from Local UI"
```

> title: "Download Files from Local UI."

Reviewer: ```suggestion
title: "Download Files from Local UI"
```

description: "Instructions for downloading files from Local UI."
hide_table_of_contents: false
sidebar_position: 130
tags: ["edge"]
---

> Palette Edge provides you with a fixed path where you can configure your applications to write files. These files can be

Reviewer: ```suggestion
Palette Edge provides you with a fixed path where you can configure your applications to write files. These files can then be
```
maybe?

downloaded from Local UI.

> This is particularly useful if you want to access core dump files of your applications in the event of a crash. If you

Reviewer: ```suggestion
This is particularly useful if you want to access core dump files of your applications in the event of a crash. For example, if you
```

have configured your applications to direct core dumps to the folder `/var/log/files` in the event of a crash, you can

> download the core dump files from Local UI for diagnostic purposes. However, you can write any file into the directory

Reviewer: ```suggestion
download the core dump files from Local UI for diagnostic purposes. You can also write any type of file to this directory; it is not limited to core dump files.
```

Reviewer: just a suggestion :)

> and they do not necessarily need to be core dump files.

Reviewer: ```suggestion
```

## Prerequisites

- You have configured your applications to write files to `/var/log/files`. The steps to do this vary by application,
  but the path must be exactly `/var/log/files`. This is the only path where files can be downloaded from Local UI.

- An operational Edge host installed in the `airgap` mode. For more information, refer to
  [Installer Reference](../../edge-configuration/installer-reference.md).

> - Network access to your Edge host's Local UI port. The default port is 5080.

Reviewer: ```suggestion
- Network access to your Edge host's Local UI port. The default port is `5080`.
```

> ## Procedure

Reviewer: ```suggestion
## Enablement
```
maybe? to follow the template
